content/docs/oracle/run-oracle/_index.md

```diff
@@ -34,62 +35,142 @@ Transaction costs for submitting price report data on-chain _are_ refunded but t
 
 1. Enter your working directory for the oracle server.
 
-2. Configure data source plugins. Navigate to the `config` sub-directory of your installation (default: `./build/bin/config`) and edit the `plugins-conf.yml` file to add plugin configuration.
+2. Create and edit your oracle server config file `plugins-conf.yml` to specify the `name` and `key` for each plugins you are using.
 
-    For how to do this see [Configure plugins](/oracle/run-oracle/#configure-plugins) on this page.
+   ```bash
+   touch plugins-conf.yml
+   ```
 
-3. (Optional) Add your own data source plugin(s). If you have developed your own FX plugins, (a) add sub-directory(ies) containing the plugin source code to the `plugins` sub-directory of your installation; (b) add config entry(ies) to the `plugin-conf.yml` file. 
+   {{< alert title="Info" >}}
+   A [sample `plugins-conf.yml` config file <i class='fas fa-external-link-alt'></i>](https://github.com/autonity/autonity-oracle/blob/master/config/plugins-conf.yml) can be downloaded from the Autonity Oracle Server GitHub.
+   {{< /alert >}}
+
+
+   Edit `plugins-conf.yml` to [configure plugins](/oracle/run-oracle/#configure-plugins) for data sources. See [Set up plugins config file](/oracle/run-oracle/#set-up-plugins-config-file) for how to do this.
+
+3. (Optional) Add your own data source plugin(s). If you have developed your own FX plugins, (a) add sub-directory(ies) containing the plugin source code to the `plugins` sub-directory of your installation; (b) add config entry(ies) to the `plugins-conf.yml` file. 
  
 4. Configure the oracle server. Specify the oracle server configuration; see [command line reference](/reference/cli/oracle/). Options can be set as system environment variables or directly in the terminal.
 
 5. Start oracle server:
 
+
     ``` bash
     autoracle \
-        -oracle_key_file="<KEYFILE>" \
-        -oracle_key_password="<PWD>" \
-        -oracle_autonity_ws_url="<WS_ADDRESS>"
-        -oracle_plugin_conf="<PLUGINS_CONFIG_FILE>/plugins-conf.yml" ;
+        -key.file="<ORACLE_KEYFILE>" \
+        -key.password="<PWD>" \
+        -ws="<WS_ADDRESS>"
+        -plugin.conf="<PLUGINS_CONFIG_FILE>/plugins-conf.yml" ;
```

Comment:
```suggestion
        -ws="<WS_ADDRESS>" \
        -plugin.conf="<PLUGINS_CONFIG_FILE>/plugins-conf.yml" \
   ;
```

Reply:
This was missing a backslash

Reply:
Added in commit b0b2cbe42f886c6b6a065832b949404f4aa216ae